src/main/java/com/google/growpod/LoadTestData.java

```diff
@@ -0,0 +1,175 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.growpod;
+
+import com.google.cloud.datastore.Batch;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery;
+import com.google.growpod.data.ContainsPlant;
+import com.google.growpod.data.Garden;
+import com.google.growpod.data.HasMember;
+import com.google.growpod.data.Plant;
+import com.google.growpod.data.User;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Helper methods to load and clear test data from any object implementing the DatastoreOptions
+ * interface.
+ */
+public class LoadTestData {
+
+  /* Datastore instance and datastore itself */
+  DatastoreOptions instance;
+  Datastore datastore;
+
+  /* List of entities available at the class-level */
+  private List<User> testUsers;
+  private List<Garden> testGardens;
+  private List<Plant> testPlants;
+  private List<HasMember> testHasMembers;
+  private List<ContainsPlant> testContainsPlants;
+
+  private void loadUsers() {
```

Comment:
There is a weird dependency here in the order that these load functions must be called. For example, `loadHasMember` must be called after `testUsers`.

Can we separate out the creation of the test data from the piece that uploads it to datastore? The former can be all static immutable data.

Reply:
Both issues you commented on in LoadTestData are holdovers from when I had the Datastore generate keys. However, now, this is no longer needed since I provide constant keys. I will make both changes.